### ISSUE TITLE
Is the protected path (default: ows/proxy) expected to be configurable in the future?

### ISSUE THREAD

**Issue by fderue** [bird-house/twitcher#36]
https://github.com/bird-house/twitcher/blob/a13dd5d783cd4410dbf7bdfedcc930a86539c14c/twitcher/utils.py#L25

It is currently set to "ows/proxy" so the function "utils.parse_service_name(url)" is working but if we have a different protected_path, it won't. So may I suggest to have something more like:

```python
def parse_service_name(url, protected_path):
    parsed_url = urlparse(url)
    service_name = None
    if parsed_url.path.startswith(protected_path):
        parts_without_protected_path = parsed_url.path[len(protected_path)::].strip('/').split('/')
        if len(parts_without_protected_path) > 0:
            service_name = parts_without_protected_path[0]
    if not service_name:
        raise ServiceNotFound
    return service_name
```

**Comment by cehbrecht** [comment]
Making the "protected path" configurable is better than having it hard-coded.

What you like to make a PR?

To configure it you need to update the `buildout.cfg` and the `twitcher.ini` template:
* https://github.com/bird-house/twitcher/blob/a13dd5d783cd4410dbf7bdfedcc930a86539c14c/buildout.cfg#L42
* https://github.com/bird-house/twitcher/blob/a13dd5d783cd4410dbf7bdfedcc930a86539c14c/templates/twitcher.ini#L20

**Comment by fderue** [comment]
Ok I will make a PR, should I do it directly to the master branch or to pingudev?

**Comment by cehbrecht** [comment]
... PR to master branch. Thanks :)

**Comment by cehbrecht** [comment]
Fixed by PR #37.
